contrib/authors_update.sh

```diff
@@ -6,32 +6,32 @@
 
 set -eEu -o pipefail
 
-range="${1?Provide commit range like orig/v1.17..orig/v1.18}"
+range="${1?Provide commit range like, for example: orig/v1.17..orig/v1.18}"
 
 if [ ! -w AUTHORS ]
 then
-    echo "AUTHORS file not accessible"
+    echo "AUTHORS file is not writable"
     exit 1
 fi
 
-if [ "$(git log --no-merges --oneline "$range" | wc -l)" -eq 0 ]
+# Failure message triggered if range is not valid
+lines=$(git log --no-merges --pretty=format:"%an%x09%ae" "$range" | sort | uniq)
```

Comment:
minor: add "-u" flag to sort instead of pipe to uniq

Reply:
fixed